### PR TITLE
KTL-763 Separate KUG and Kotlin Night guidelines

### PR DESCRIPTION
**Pull request by nikpachoo**
https://youtrack.jetbrains.com/issue/KTL-763/Separate-KUG-and-Kotlin-Night-guidelines

### CODE REVIEW

File: docs/topics/kotlin-nights-branding.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[//]: # (title: Kotlin Nights branding)
+[//]: # (title: Kotlin Nights Guidelines)
```
Comment (koshachy) [inline docs/topics/kotlin-nights-branding.md:1]:
```suggestion
[//]: # (title: Kotlin Nights guidelines)
```

File: docs/topics/kug-branding.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[//]: # (title: KUG branding)
+[//]: # (title: KUG Guidelines)
```
Comment (koshachy) [inline docs/topics/kug-branding.md:1]:
```suggestion
[//]: # (title: KUG guidelines)
```

**Review by koshachy — APPROVED** [review]

File: docs/topics/kotlin-nights-branding.md (hunk 1 above)
Comment (meilalina) [inline docs/topics/kotlin-nights-branding.md:1]: @nikpachoo can we please have Kotlin Night Guidelines instead?

Comment (koshachy) [reply docs/topics/kotlin-nights-branding.md:1]: We don't use title case at kotlinlang.org/docs

File: docs/kr.tree [hunk 3]
```diff
@@ -362,13 +361,15 @@
...
-            <toc-element toc-title="Press kit" href="https://kotlinlang.org/assets/kotlin-media-kit.pdf"/>
+        <toc-element toc-title="Community">
+            <toc-element id="contribute.md"/>
+            <toc-element id="kug-guidelines.md"/>
+            <toc-element id="kotlin-night-guidelines.md"/>
         </toc-element>
+
+        <toc-element id="kotlin-brand-assets.md"/>
+        <toc-element id="kotlin-mascot.md" hidden="true"/>
+        <toc-element toc-title="Press kit" href="https://kotlinlang.org/assets/kotlin-media-kit.pdf"/>
```
[the remark this answers is not in the document]
Comment (koshachy) [reply docs/kr.tree:371]: Disscused in Slack that will return to it later.

File: redirects/pages.yml [hunk 4]
```diff
@@ -668,4 +668,13 @@
 - from:
     - /docs/kmm-configure-sqldelight-for-data-storage.html
     - /docs/multiplatform-mobile-configure-sqldelight-for-data-storage.html
-  to: https://play.kotlinlang.org/hands-on/Networking%20and%20Data%20Storage%20with%20Kotlin%20Multiplatfrom%20Mobile/
\ No newline at end of file
+  to: https://play.kotlinlang.org/hands-on/Networking%20and%20Data%20Storage%20with%20Kotlin%20Multiplatfrom%20Mobile/
+
+- from: /docs/kotlin-nights-branding.md
```
Comment (koshachy) [inline redirects/pages.yml:673]: I think, that WH will be enough, or it's better to writer them here explicitly?

Comment (nikpachoo) [reply redirects/pages.yml:673]: You are right, i'll fix it.

**Review by koshachy — APPROVED** [review]
LGTM

**Pull request merged**